pkg/driver/http_client.go

```diff
@@ -57,6 +57,7 @@ func InvokeOpenWhisk(function *common.Function, runtimeSpec *common.RuntimeSpeci
 	log.Tracef("(Invoke)\t %s: %d[ms], %d[MiB]", function.Name, runtimeSpec.Runtime, runtimeSpec.Memory)
 
 	success, executionRecordBase, res := httpInvocation("", function, AnnounceDoneExe, true)
+	AnnounceDoneExe.Wait() // Refer to Issue 329: https://github.com/vhive-serverless/invitro/issues/329
```

Comment:
Better to add an explanation for that from the comment in the issue instead. Like 'To postpone querying OpenWhisk during the experiment for performance reasons'.